subworkflows/MODEL_TO_DATA.nf

```diff
@@ -44,14 +45,15 @@ workflow MODEL_TO_DATA {
         .splitCsv(header:true) 
         .map { row -> tuple(row.submission_id, row.image_id) }
     UPDATE_SUBMISSION_STATUS_BEFORE_RUN(image_ch.map { tuple(it[0], "EVALUATION_IN_PROGRESS") })
-    CREATE_FOLDERS(image_ch.map { tuple(it[0], "build") }, params.project_name, UPDATE_SUBMISSION_STATUS_BEFORE_RUN.output)
-    RUN_DOCKER(image_ch, SYNAPSE_STAGE.output, params.cpus, params.memory, CREATE_FOLDERS.output)
-    UPDATE_SUBMISSION_STATUS_AFTER_RUN(RUN_DOCKER.output.map { tuple(it[0], "ACCEPTED") })
-    VALIDATE(RUN_DOCKER.output, UPDATE_SUBMISSION_STATUS_AFTER_RUN.output, params.validation_script)
-    UPDATE_SUBMISSION_STATUS_AFTER_VALIDATE(VALIDATE.output.map { tuple(it[0], it[2]) })
-    ANNOTATE_SUBMISSION_AFTER_VALIDATE(VALIDATE.output)
-    SCORE(VALIDATE.output, UPDATE_SUBMISSION_STATUS_AFTER_VALIDATE.output, ANNOTATE_SUBMISSION_AFTER_VALIDATE.output, params.scoring_script)
-    UPDATE_SUBMISSION_STATUS_AFTER_SCORE(SCORE.output.map { tuple(it[0], it[2]) })
-    ANNOTATE_SUBMISSION_AFTER_SCORE(SCORE.output)
-    SEND_EMAIL(params.view_id, image_ch.map { it[0] }, params.email_with_score, ANNOTATE_SUBMISSION_AFTER_SCORE.output)
+    RUN_DOCKER(image_ch, SYNAPSE_STAGE.output, params.cpus, params.memory, UPDATE_SUBMISSION_STATUS_BEFORE_RUN.output)
+    CREATE_FOLDERS(image_ch.map { tuple(it[0], "build") }, params.project_name, RUN_DOCKER.output.map { it[1] }, "ready")
```

Comment:
Feeding the CREATE_FOLDERS process the `RUN_DOCKER.output.map { it[1] }` despite it not being used in the script call does no harm, but looks misleading. Find a more elegant way to write this. Can a `path` input be a conditional? i.e. "Ignore `path` input if == null or "None" etc"